Presentation/UINavigationController+Presenting.swift

```diff
@@ -11,16 +11,16 @@ import Flow
 
 public extension PresentationOptions {
     /// Pushing and popping on a navigation controller defaults to batch subsequent operation from the same run-loop togeher. This options turnes that of.
-    public static let disablePushPopCoalecing = PresentationOptions()
+    static let disablePushPopCoalecing = PresentationOptions()
```

Comment:
_Coalecing_ should be _Coale**s**cing_, but I'm not sure fixing this typo is worth all the headache it involves.